command_line/cosym.py

```diff
@@ -177,9 +178,17 @@ def run(self):
         self.cosym_analysis.run()
 
         reindexing_ops = {}
+        sym_op_counts = {
+            cluster_id: collections.Counter(
```

Comment:
Wait, what is this syntax?

Learning opportunity for YT

Reply:
Oh, dictionary creation from iterator...

Reply:
Possibly re-arranging the code slightly (e.g. a dummy variable to move everything onto a single line) would make it look less magic?